test_scripts/Defects/8_0/3783_crash_during_vr_langchange.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/3783
+---------------------------------------------------------------------------------------------------
+-- Description: A crash occurs during registration (occurance: about 1%)
+--
+-- Steps:
+-- 1. Change VR language to FR-CA
+-- 2. Connect EN-US apps
+-- 4. Quickly register 5 apps and then unregister them
+-- 3. Change VR language to FR-CA
+-- 4. Quickly register 5 apps and then unregister them
+-- SDL does:
+--  - not crash
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+local utils = require("user_modules/utils")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+
+--[[ Local Functions ]]
+local function registerApp(pAppId)
+    local session = common.getMobileSession(pAppId, 1)
+    session:StartService(7)
+    :Do(function()
+        local corId = session:SendRPC("RegisterAppInterface", common.app.getParams(pAppId))
+        common.hmi.getConnection():ExpectNotification("BasicCommunication.OnAppRegistered",
+          { application = { appName = common.app.getParams(pAppId).appName } })
+        :Do(function(_, d1)
+            common.app.setHMIId(d1.params.application.appID, pAppId)
+          end)
+        session:ExpectResponse(corId, { success = true })
+        :Do(function()
+            session:ExpectNotification("OnHMIStatus",
+              { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+            session:ExpectNotification("OnPermissionsChange")
+            :Times(AnyNumber())
+          end)
+      end)
+  end
+
+local function changeHMILanguage(iface, lang)
+  local hmi = common.hmi.getConnection()
+  hmi:SendNotification(iface .. ".OnLanguageChange", { language = lang })
+  utils.wait(500)
+end
+
+local function registerAllApps()
+  for i = 1,5 do
+    common.getMobileSession(i, 1):SendRPC("RegisterAppInterface", common.app.getParams(i))
+  end
+  utils.wait(250)
+  for j = 1,5 do
+    common.getMobileSession(j):SendRPC("UnregisterAppInterface", {})
+  end
```

Comment:
For this issue specifically, I don't think we are supposed to manually unregister the apps, they should be unregistered automatically due to the language change

Reply:
I have removed extraneous steps from the test in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2603/commits/3acb6fd631731f53df8aa6ece82d4744f42d30ae